api/yolopb.proto

```diff
@@ -141,6 +148,11 @@ message Build {
 
   /// relationships
 
+  string raw_branch = 21;
+  string raw_commit = 22;
```

Comment:
```suggestion
  Commit raw_commit = 22;
```

Reply:
the goal is to use those fields in the build list with gorm's Preload helper.

Reply:
@moul we can't override these infos so why should we add (hasRawProject & hasRawCommit) ?

Reply:
so that we can do `.Preload("RawCommit")` with gorm.